src/algebra/order_functions.lean

```diff
@@ -29,8 +29,8 @@ variables [linear_order α] [linear_order β] {f : α → β} {a b c d : α}
 @[simp] lemma max_le_iff : max a b ≤ c ↔ a ≤ c ∧ b ≤ c := sup_le_iff
 lemma max_le_max : a ≤ c → b ≤ d → max a b ≤ max c d := sup_le_sup
 lemma min_le_min : a ≤ c → b ≤ d → min a b ≤ min c d := inf_le_inf
-lemma le_max_left_of_le : a ≤ b → a ≤ max b c := le_sup_left_of_le
-lemma le_max_right_of_le : a ≤ c → a ≤ max b c := le_sup_right_of_le
+lemma le_max_of_le_left : a ≤ b → a ≤ max b c := le_sup_of_le_left
+lemma le_max_of_le_right : a ≤ c → a ≤ max b c := le_sup_of_le_right
 lemma min_le_left_of_le : a ≤ c → min a b ≤ c := inf_le_left_of_le
```

Comment:
Don't you think the `min` version would also deserve a renaming, like `min_le_of_left_le` and friends?